README.md

```diff
@@ -26,54 +38,79 @@ https://hub.docker.com/_/nats/
 You can find the instructions for the NATS Cli here:
 https://github.com/nats-io/natscli
 
+A `docker-compose` file is also available for standing up NATS Jetstream server.
+
 ### Create the HL7 stream and MESSAGES consumer
 
 Create the HL7 stream with wildcard/unspecified subjects:
 
 ```bash
-nats str add HL7 --subjects "HL7.*" --ack --max-msgs=-1 --max-bytes=-1 --max-age=1y --storage file --retention limits --max-msg-size=-1 --discard=old --max-msgs-per-subject=-1 --dupe-window=2m --replicas=1
+nats str add hl7 --subjects "hl7.*" --ack --max-msgs=-1 --max-bytes=-1 --max-age=1y --storage file --retention limits --max-msg-size=-1 --discard=old --max-msgs-per-subject=-1 --dupe-window=2m --replicas=1
 ```
 
 Create the MESSAGES consumer for the HL7 stream and filter to a subject named "HL7.MESSAGES":
```

Comment:
Nit – not calling it `MESSAGES` anymore right?

Reply:
Fixed.